buildSrc/src/main/groovy/io.micronaut.build.internal.liquibase.test-suite-module.gradle

```diff
@@ -1,32 +1,29 @@
 plugins {
     id("groovy")
-    id("io.micronaut.library")
 }
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    testImplementation(project(":liquibase"))
-
-    testImplementation("io.micronaut:micronaut-http-validation")
-    testImplementation("io.micronaut:micronaut-http-client")
-
+    testImplementation(platform("io.micronaut:micronaut-bom:$micronautVersion"))
+    testImplementation("io.micronaut.groovy:micronaut-runtime-groovy")
```

Comment:
```suggestion
```

` testImplementation("io.micronaut.groovy:micronaut-runtime-groovy")` is duplicated.